src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js

```diff
@@ -11,16 +13,30 @@ export class DocumentEItems extends Component {
     element.scrollIntoView({ behavior: 'smooth' });
   };
 
+  onClickEItemRequestLink = () => {
+    const medium = 'ELECTRONIC';
+    return {
+      pathname: FrontSiteRoutes.documentRequestForm,
+      state: { medium },
+    };
+  };
+
   render() {
     const { eitems } = this.props;
 
-    return eitems.total > 0 ? (
+    return (
       <>
         <Header as="h3">Access online</Header>
-        <DocumentLinks eitems={eitems} />
+        {eitems.total > 0 ? (
+          <DocumentLinks eitems={eitems} />
+        ) : (
+          <Link to={this.onClickEItemRequestLink()}>
+            Request an electronic item.
```

Comment:
Looks odd to me; I'm suggesting something like that:
`No resource available. <a>Submit a request</a>`